notebooks/official/generative_ai/rlhf_tune_llm.ipynb

```diff
@@ -946,7 +934,7 @@
         "id": "TlICJjNqvJlx"
       },
       "source": [
-        "You can also interact with your model using Generative AI Studio. Click on **Deploy Model** > **Output Parameters** > `endpoint_resource_name` hyperlink. Then click on the model name > \"Deploy & Test\" tab > \"Open in Prompt Design\"."
+        "You can also interact with your model using Generative AI Studio. Click on **Deploy Model** > **Output Parameters** > `endpoint_resource_name` hyperlink. Then click on the model name > **Deploy & Test** tab > **Open in Prompt Design**."
```

Comment:
Remove "on" in two cases:

- change: "Click on" 
- to: "Click:

- change: "click on"
- to: "click"